.github/workflows/python-package.yml

```diff
@@ -21,18 +21,24 @@ jobs:
                 astropy-version: ['==5.0']  # fuji+guadalupe version
                 fitsio-version: ['==1.1.6']  # fuji+guadalupe version
                 numpy-version: ['<1.23']  # to keep asscalar, used by astropy
+                include:
+                  - os: ubuntu-latest
+                    python-version: '3.10'
+                    astropy-version: '<7'
+                    fitsio-version: '==1.2.4'
+                    numpy-version: '==2.0.2'
```

Comment:
Naive question: I'm confused about these versions being specified at two different levels in the test config hierarchy (jobs.tests.strategy.matrix.python-version vs. jobs.tests.strategy.matrix.include[0].python-version).  I was expecting something like a list with two elements specifying two different sets of versions.  What motivates this structure, and how would it adapt if we wanted a 3rd set of versions?

Reply:
I'm not an expert on this, but I've checked the github docs, and that's how one adds one single extra configuration on top of the existing Cartesian product (which ATM contains 1 config).
One can restructure this as two distinct configurations.